src/AlgebraicGeometry/Schemes/AffineAlgebraicSet/Objects/Constructors.jl

```diff
@@ -86,17 +86,17 @@ julia> (x, y) = coordinates(A)
 julia> X = algebraic_set(ideal([y - x^2]))
 Affine algebraic set
   in affine 2-space over QQ with coordinates [x, y]
-defined by ideal(-x^2 + y)
+defined by Ideal with 1 generator in multivariate polynomial ring
```

Comment:
Another situation where the old printing seems strictly more useful than the new printing :-/.

Of course that might change if there are 50 ideal generators, each spanning three lines...


BTW, there seems to be a `Lowercase()` missing in the show method for `Affine algebraic set`

Reply:
I agree that for a few, small generators the old printing is better. In the end this gets us back at the discussion, whether objects should print all their generators or not or just some.
Here, I followed the example in #2166 by @simonbrandhorst , but we could of course also print something like
```
Ideal generated by x1, x2, ..., xn in multivariate polynomial ring
```
in one-line mode. If one generator is just huge this will of course explode again.